explainer.md

```diff
@@ -1,624 +1,397 @@
+
+Parts don't need to be associated with another part, which allows for transitions involving elements that are only in Page-A or only in Page-B.
+
+The root parts of each page are automatically associated.
 
-### New Document
+## Part 3: The transition
+
+Everything is now in place to perform the transition. The developer can move the parts around using the usual APIs, such as CSS and web animations.
+
+https://user-images.githubusercontent.com/93594/141100217-ba1fa157-cd79-4a9d-b3b4-67484d3c7dbf.mp4
+
+## Part 4: The end
+
+When the transition is complete, the top level is removed, revealing the real Page-B.
+
+# The API
+
+There are a lot of open questions around the API, but this section will give a flavour of the direction.
+
+Little thought has been giving to naming in these APIs, which will be improved when it's clearer what the right direction is.
+
+Additionally, this API focused on cross-document transitions.
+
+## Creating a transition part
+
+There are a few ideas for how a developer could offer an element as a transition part.
+
+### Attributes
 
 ```html
-<html>
-<head>
-  <style>
-    body {
-      background-color: blue;
-    }
-
-    .animated {
-      position: relative;
-      top: 100px;
-      width: 200px;
-      height: 200px;
-      background-color: green;
-
-      border: 5px solid black;
-      border-radius: 50% 50%;
-      box-shadow: 0px 0px 5px;
-    }
-
-    .animated:transition {
-      border: 5px solid transparent;
-      border-radius: none;
-      box-shadow: none;
-    }
-
-    ::shared-container(header-id) {
-        border: 5px solid black;
-        border-radius: 50% 50%;
-        box-shadow: 0px 0px 5px;
-    }
-
-    ::shared-new(header-id) {
-      top: -5px;
-      left: -5px;
-    }
-
-    @keyframes border-animation {
-      from {
-        border: 10px solid black;
-        border-radius: 10% 10%;
-        box-shadow: 0px 0px 10px;
-      }
-    }
-    ::shared-container(header-id) {
-      animation: ::shared-container-header-id 1s ease-in, border-animation 1s ease-in;
-    }
-
-    @keyframes position-animation {
-      from {
-        top: -10px;
-        left: -10px;
-      }
-      to {
-        top: -5px;
-        left: -5px;
-      }
-    }
-    ::shared-old(header-id), ::shared-new(header-id) {
-      animation: position-animation 1s ease-in;
-    }
-
-  </style>
-</head>
-<body>
-  <div class="animated" sharedid="header-id" id="header">Shared Element</div>
-</body>
-</html>
+<header pagetransitiontag="header" pagetransitioncapture="container-and-child">…</header>
 ```
 
-The additional steps taken by the browser in the example above are :
+Here, the presence of the `pagetransitiontag` attribute marks this element as a transition part, and the `pagetransitioncapture` attribute is used to indicate how it's captured (as a single texture vs as a container + child texture).
```

Comment:
It seems like the page-transition-tag css property is strictly superior to the pageTransitionTag attribute, do we need both? If we have both what does it mean if they have different values?

Reply:
Yeah, it's never going to be both. We need to pick one. I prefer CSS properties too.